spring-boot-project/spring-boot-docs/src/main/asciidoc/appendix-application-properties.adoc

```diff
@@ -1365,6 +1365,8 @@ content into your application. Rather, pick only the properties that you need.
 	management.metrics.binders.processor.enabled=true # Whether to enable processor metrics.
 	management.metrics.binders.uptime.enabled=true # Whether to enable uptime metrics.
 	management.metrics.distribution.percentiles-histogram.*= # Whether meter IDs starting with the specified name should publish percentile histograms.
+	management.metrics.distribution.minimum-expected-value.*= # Minimum limit on the histogram buckets for IDs starting with the specified name. The longest match wins, the key `all` can also be used to configure all meters.
```

Comment:
May or may not be worth mentioning that minimum/maximum can affect the accuracy of client-side percentiles if not set to realistic values.

Reply:
I'm not sure how to phrase it in a useful way, and the docs already invite the reader to check Micrometer documentation for more details.

Reply:
I don't think we can support `all` here. A `Duration` can specify a unit, for example `30s`. How would that be converted to a plain `long` for those meters that require that rather than a `Duration`?

Reply:
We've decided to drop support for `sla` so we shouldn't add support for all here either.

Reply:
Dropping `sla` is a usability concern, as this is going be a frequently set property. What is the reasoning behind this decision?

Reply:
We’re not dropping support for `sla` as a whole, just `all` as it doesn’t make sense given the mixture of long and duration values depending on the meter type.

Reply:
Oh, I see. Many thanks.